app/services/BreathingSpaceService.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import cats.implicits._
+import com.google.inject.Inject
+import config.ConfigDecorator
+import connectors.BreathingSpaceConnector
+import uk.gov.hmrc.domain.Nino
+import uk.gov.hmrc.http.HeaderCarrier
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class BreathingSpaceService @Inject() (
+  configDecorator: ConfigDecorator,
+  breathingSpaceConnector: BreathingSpaceConnector
+) {
+
+  def getBreathingSpaceIndicator(
+    ninoOpt: Option[Nino]
+  )(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Boolean] =
```

Comment:
Boolean may not be the best return type. A tax payer can actually be in 1 of 3 states
1. User is in a breathing space period - true response from api
2. User is not in a breathing space period - false response from api
3. User does not have breathing space - NotFound response from api

If the business decide they want to inform a user when their breathing space period has ended this code could introduce non obvious bugs. My suggestion would be to create a type that represents the three states. Something like:

```
sealed trait BreathingSpaceIndicator
object BreathingSpaceIndicator {
  case object WithinPeriod extends BreathingSpaceIndicator // true status
  case object OutOfPeriod extends BreathingSpaceIndicator // false status
  case object NotFound extends BreathingSpaceIndicator // not found response
  case object StatusUnknown extends BreathingSpaceIndicator // all others including feature disabled and exceptions
  
  def fromBoolean(b: Boolean): BreathingSpaceIndicator =
    if(b) WithinPeriod else OutOfPeriod
}
```

Example usage below

Reply:
Fixed , Please re-review , thanks